app/pages/Contacts/ContactsPage.presenter/ContactsPage.presenter.tsx

```diff
@@ -3,16 +3,21 @@ import type { FC } from 'react';
 
 import { useSnackbar } from 'notistack';
 import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
 import { Redirect } from 'react-router-dom';
 
 import { randomColor } from '../../../constants/app';
-import useFullService from '../../../hooks/useFullService';
-import { assignAddressForAccount, updateContacts } from '../../../services';
+import { ReduxStoreState } from '../../../redux/reducers/reducers';
+import { updateContacts } from '../../../redux/services';
+import { assignAddressForAccount } from '../../../services';
+import { SelectedAccount } from '../../../types';
 import type { Contact } from '../../../types/Contact.d';
 import { ContactForm } from '../ContactForm.view';
 import { ContactsList } from '../ContactsList.view';
 
-const ContactsPage: FC = () => {
+type Props = ReduxProps;
```

Comment:
done for consistency's sake. where component props are needed, this becomes `type Props = ElementProps & ReduxProps;`